packages/shared/test-suites/RecordingTestMetadata.ts

```diff
@@ -412,7 +416,9 @@ export async function processCypressTestRecording(
 
         const annotations = userActionEventIdToAnnotations[id];
 
-        assert(annotations != null, `Missing annotations for test event (${command.name})`);
+        assert(annotations != null, `Missing annotations for test event`, {
+          command: command.name,
+        });
```

Comment:
We should probably add the id here too, as is done with the others.
```suggestion
        assert(annotations != null, `Missing annotations for test event`, {
          command: command.name,
          id
        });
```